src/FakeItEasy/Creation/IFakeCreatorFacade.cs

```diff
@@ -28,6 +37,16 @@ internal interface IFakeCreatorFacade
         T CreateDummy<T>();
 
         /// <summary>
+        /// Creates a dummy object, this can be a fake object or an object resolved
+        /// from a dummy factory.
+        /// </summary>
+        /// <param name="typeOfDummy">The type of dummy to create.</param>
+        /// <returns>The created dummy.</returns>
+        /// <exception cref="FakeItEasy.Core.FakeCreationException">Was unable to generate the fake in the current configuration and
+        /// no dummy was registered in the container for the specified type..</exception>

```

Comment:
The double "." is odd here, but the docs were just copied, so not your fault. At least it's consistent.